code/game/objects/effects/effect_system.dm

```diff
@@ -27,6 +28,10 @@ would spawn and follow the beaker, even if it is carried or thrown.
 			M.Turn(rand(0,360))
 
 		transform = M
+	if(random_offset)
```

Comment:
Shouldn't that be in `Initialize`?

Reply:
is there a solid reason why it should be in Initialize ?

Reply:
All initialization code should go into `Initialize`/`LateInitialize` to prevent maploading lag and keep things controllable.

Reply:
fixed